recipes/cylc-flow/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "cylc-flow" %}
+{% set version = "8.0a1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "36f4f4363cb5cc7a3a613d2db6e84cc41c2825c6cd011191cc4821c5ca9c81a4"
+
+build:
+  number: 0
+  skip: True  # [not linux]
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
+requirements:
+  host:
+    - ansimarkup >=1.0.0
+    - colorama >=0.4
+    - graphene ==2.1.6
+    - jinja2 >=2.10.1
+    - markupsafe >=1.1
+    - metomi-isodatetime >=1!2.0.1
+    - pip
+    - protobuf >=3.7
+    - pytest-runner
+    - python >=3.7
+    - python-jose >=3.0
+    - pyzmq >=18.0
+  run:
+    - ansimarkup >=1.0.0
+    - colorama >=0.4
+    - cryptography
+    - graphene ==2.1.6
+    - jinja2 >=2.10.1
+    - markupsafe >=1.1
+    - metomi-isodatetime >=1!2.0.1
+    - protobuf >=3.7
+    - pyasn1
+    - python >=3.7
+    - python-jose >=3.0
+    - pyzmq >=18.0
+
+test:
+  imports:
+    - cylc.flow
```

Comment:
Any reason for removing the test command? The imports tests that the module is correctly added to python path. But the test for commands confirms the cylc command was added to the path too?

Reply:
Oh, one of the DevOps CI builds implied that the ``cylc`` command could not be imported, & looking at a selection of other recipes under ``conda-forge/feedstocks`` commands from the package module itself were not used, so I thought I would see what happened without this for debugging, after which I could re-add it. (I did not think you would be awake at this late NZ hour, but planned to sanity check the change with you afterwards!)

It looks like we pass all checks without that metadata item, so I'll now add it back in & hopefully it will all still work (it may have just been failing due to trying to build for Windows).

Reply:
If it fails the build we can remove it for now. As long as `conda build .` followed by `conda install --use-local cylc-flow` correctly install it I think :+1:

Reply:
Just squashed it in now, let's see what happens! Thanks for the advice.